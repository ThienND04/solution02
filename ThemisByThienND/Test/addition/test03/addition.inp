10
4346 8173 2155 999992162
3218 4402 1217 999996677
2667 8393 1311 999999719
3940 8320 2309 999996266
2024 6427 1972 999996967
3643 8183 2271 999996986
3857 4795 2068 999990720
1392 7107 1185 999991523
4101 8419 1790 999998127
3779 5049 2331 999997284
