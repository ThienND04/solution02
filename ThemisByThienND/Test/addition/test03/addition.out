6532074
29003057
17038402
6605726
19491666
7385581
13072969
0
8090425
11241942
